packages/opentelemetry-core/test/common/logging-error-handler.test.ts

```diff
@@ -24,11 +24,8 @@ describe('loggingErrorHandler', () => {
 
   beforeEach(() => {
     handler = loggingErrorHandler({
-      debug: sinon.fake(),
-      info: sinon.fake(),
-      warn: sinon.fake(),
       error: errorStub,
-    });
+    } as any);
```

Comment:
This is possible because the base loggers (api.diag and logLevelDiagLoger) perform checks on whether the called function exists -- so this is safe from a testing perspective.

Reply:
it's in a test anyway